cluster/examples/kubernetes/ceph/upgrade-from-v0.8-create.yaml

```diff
@@ -214,6 +214,20 @@ metadata:
   labels:
     operator: rook
     storage-backend: ceph
+aggregationRule:
```

Comment:
Don't worry about updating the upgrade-from-v0.8 yaml files. @BlaineEXE is working on the 0.9 to 1.0 upgrade guide, which will replace these files.